apiserver/Dockerfile.hail-base

```diff
@@ -6,8 +6,8 @@ RUN unzip -q hail.zip && \
   mkdir -p /hail/python && \
   mv inner /hail/python/hail
 
-COPY hail-all-spark.jar /hail/jars/hail-all-spark.jar
-RUN cp /hail/jars/hail-all-spark.jar $SPARK_HOME/jars/hail-all-spark.jar
+COPY hail.jar /hail/jars/hail.jar
```

Comment:
Does the renaming of the jar affect cloud tools or the deployment steps at all?

Reply:
Yes.  I renamed things throughout to be hail.jar.